src/NuGet.Server/Infrastructure/ServerPackage.cs

```diff
@@ -79,8 +39,10 @@ public ServerPackage(IPackage package, PackageDerivedData packageDerivedData)
             MinClientVersion = package.MinClientVersion;
             ReportAbuseUrl = package.ReportAbuseUrl;
             DownloadCount = package.DownloadCount;
-            IsAbsoluteLatestVersion = package.IsAbsoluteLatestVersion;
-            IsLatestVersion = package.IsLatestVersion;
+            SemVer1IsAbsoluteLatest = false;
+            SemVer1IsLatest = false;
+            SemVer2IsAbsoluteLatest = false;
+            SemVer2IsLatest = false;
```

Comment:
This is essentially changing the `IsLatest` and `IsAbsoluteLatest` from a stored property to a computed property on the `ServerPackage` type (having it computed for `ODataPackage` is what we have today but it is done so without additional calls to the repository layer). I don't think this would be much simpler since the means the `IServerPackage` instances returned from the repository would have to be re-initialized or cloned for each API call with respect to the `SemVerLevel` parameter.

Alternatively (and alluded to above), we have an a API on the repository to discover the `IsLatest` properties for `ODataPackage` instances after they have been mapped from `ServerPackage`. I also think this isn't as clear since the controller layer will now have even more logic, i.e. more necessary calls to the repository layer that must be done in order to gave good data.

Unless you guys feel strongly, I prefer keeping this implementation so that View and Model concerns are separated.